Src/FluentAssertions/Types/MethodBaseAssertions.cs

```diff
@@ -93,8 +93,6 @@ public AndConstraint<TAssertions> NotHaveAccessModifier(CSharpAccessModifier acc
             return new AndConstraint<TAssertions>((TAssertions)this);
         }
 
-        protected override string Identifier => "methodBase";
```

Comment:
`MethodBaseAssertions` is `abstract` and its two derivatives `ConstructorInfoAssertions` and `MethodInfoAssertions` overrides `Identifier` with `"constructor"` and `"method"`, respectively.